### ISSUE TITLE
Implement original hunter-seeker targeting logic.

### ISSUE THREAD

**Issue by pchote** [OpenRA/OpenRA#12918]
This is distinct from #6478, which is something that we probably want to consider as a future gameplay improvement after we have a properly working game.

**Comment by pchote** [comment]
I believe the original logic was to pick an enemy actor at random?

**Comment by reaperrr** [comment]
Yes. Which is why many loathed that system, but I guess just as many would complain if we didn't implement it.

**Comment by GraionDilach** [comment]
I believe this would be best resolved via using the current AttackLeap system (not the proposed one) with a random target chosen at launch.
